netplugin/agent/docker_event.go

```diff
@@ -28,64 +28,80 @@ import (
 	"golang.org/x/net/context"
 )
 
-// Handles docker events monitored by dockerclient. Currently we only handle
-// container start and die event*/
+// Handles docker events monitored by dockerclient. Currently we only handle // container start and die event*/
 func handleDockerEvents(event *dockerclient.Event, ec chan error, args ...interface{}) {
 
 	log.Debugf("Received Docker event: {%#v}\n", *event)
 	endpointUpdReq := &master.UpdateEndpointRequest{}
-	switch event.Status {
-	case "start":
+
+	var containerID string
+	var networkID string
+
+	if (event.Type == "container" && event.Action == "start") ||
```

Comment:
These are docker specific events. Better to have the same constants that docker uses if they are available(like ContainerEventType)